### PR TITLE
Release v1.5.0

### PR DESCRIPTION
**Pull request by gtxu**
**Is this a bug fix or adding new feature?**

**What is this PR about? / Why do we need it?**

**What testing is done?**

### CODE REVIEW

File: deploy/kubernetes/overlays/stable/ecr/kustomization.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ bases:
 images:
   - name: k8s.gcr.io/provider-aws/aws-ebs-csi-driver
     newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-ebs-csi-driver
+    newTag: v1.5.0
```
Comment (wongma7) [inline deploy/kubernetes/overlays/stable/ecr/kustomization.yaml:8]: also update the new gcr overlay tag (it was just merged yesterday )

Comment (gtxu) [reply deploy/kubernetes/overlays/stable/ecr/kustomization.yaml:8]: Done

File: charts/aws-ebs-csi-driver/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: 1.4.0
+appVersion: 1.5.0
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
 version: 2.4.1
```
Comment (wongma7) [inline charts/aws-ebs-csi-driver/Chart.yaml:5]: should also bump this to 2.5.0 and update chart changelog

Comment (gtxu) [reply charts/aws-ebs-csi-driver/Chart.yaml]: This is already bumped by v2.5.0 via post-release. May reversed by merge. Update done know.

**Comment by wongma7** [comment]
/lgtm
/approve

thank you

**Comment by gtxu** [comment]
Thank you for reviewing @wongma7 !

**Comment by gtxu** [comment]
Solve #1128

**Pull request merged**